lib/time/duration.ex

```diff
@@ -424,12 +545,25 @@ defmodule Timex.Duration do
   end
 
   @doc """
-  Time interval since Epoch.
+  Returns the amount of time since Epoch.
+
+  The argument is an atom indicating the type of time units to return.
+
+  The allowed unit type atoms are:
+  - :microseconds
+  - :milliseconds
+  - :seconds
+  - :minutes
+  - :hours
+  - :days
+  - :weeks
+
+  ## Examples
 
-  The argument is an atom indicating the type of time units to return (see
-  convert/2 for supported values).
+  iex> Timex.Duration.now(:seconds)
+  :os.system_time(:seconds)
 
-  When the argument is omitted, the return value's format is { megasecs, seconds, microsecs }.
+  When the argument is omitted or nil, a Duration is returned.
```

Comment:
Same as above, I think the best test is one which verifies that the value returned is valid for the current date/time, i.e. it's not above or below a range which is reasonable for covering the inherent race condition present in the function.